frontend/src/components/RegionDropdown.js

```diff
@@ -17,6 +17,8 @@ function RegionDropdown({
         {REGIONS.map(({ number, name: description }) => (
           <option key={number} value={number}>{`${number} - ${description}`}</option>
         ))}
+        {includeCentralOffice
+        && <option name="central-office" value="13">Central Office</option>}
```

Comment:
Might be better to treat the values as strings and use `CO` for Central Office. `13` is sometimes used for a special region value of grantees with major monitoring issues.